package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dvf-client-js",
-  "version": "2.8.3",
+  "version": "3.0.0",
```

Comment:
is changing `dev` blockchain to goerli a major breaking change of the client?

Reply:
It's a good question. @Matko95 what do you think ?

Reply:
@arijoon It probably is for everyone integrating the library